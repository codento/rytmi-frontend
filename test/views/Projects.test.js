import Vuex from 'vuex'
import { shallow, createLocalVue } from 'vue-test-utils'
import View from '../../src/views/Projects.vue'

describe('Profile.vue shallow', () => {
  let localVue = createLocalVue()
  localVue.use(Vuex)
  let store, cmp

  beforeEach(() => {
    cmp = shallow(View, {
      store,
      localVue
    })
  })
  it('Template is correct', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
