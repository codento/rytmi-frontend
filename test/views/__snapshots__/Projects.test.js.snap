// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile.vue shallow Template is correct 1`] = `
<div
  class="animated fadeIn profile-editor"
>
  <!---->
   
  <!---->
</div>
`;
